queries/prolog/highlights.scm

```diff
@@ -0,0 +1,59 @@
+(comment) @comment
+
+(atom) @constant
+
+((atom) @boolean
+  (#eq? @boolean "true"))
+
+((atom) @boolean
+  (#eq? @boolean "false"))
+
+(functional_notation
+  function: (atom) @function.call)
+
+[
+  (integer)
+  (negative_number
+    (integer))
+] @number
+
+[
+  (float_number)
+  (negative_number
+    (float_number))
+] @number.float
+
+[
+  (operator_1200xfx)
+  (operator_1200fx)
+  (operator_1100xfy)
```

Comment:
1100 (semicolon) should be moved to `@punctuation.delimiter`. To be honest I think most or all of the `operator_*` nodes should be prefixed with underscores in the grammar, these node names seem pretty unintuitive :sweat_smile:

Reply:
Unlike periods, commas and semicolons are considered operators in Prolog, but I can see how thinking about them as punctuation would be more intuitive, so I'll change this.

I thought hiding nodes would prevent me from referring to them in queries?

Reply:
That would be a great compromise, thanks! Also if things like semicolons are meant to be operators then it's fine to keep it that way; my mistake :)

Reply:
Whoopsie I hadn't seen this commit in time and committed already. It's okay, I like it better your way anyway.